frontend/src/AudioAnnotator/AudioAnnotator.js

```diff
@@ -129,6 +130,10 @@ class AudioAnnotator extends Component<AudioAnnotatorProps, AudioAnnotatorState>
       taskStartTime: now.getTime(),
       annotations: [],
       currentDefaultTagAnnotation: '',
+      alphanumeric_keys :[
+        ["&", "é", "\"", "'", "(", "-", "è", "_", "ç"],
+        ["1", "2", "3", "4", "5", "6", "7", "8", "9"]
```

Comment:
This is not cross-keyboard, no ? :sweat_smile: like will it work on qwerty ?

Reply:
Well it's a bit too French-centric for my taste, I guess this could do for now :thinking:. @Project-ODE/researchers should we also try to add top-qwerty-keyboard keys ?

Reply:
I can add top qwerty keyboard keys if you want but both will be activated at the same time.